dojo/templates/dojo/view_finding.html

```diff
@@ -236,6 +244,9 @@ <h3 class="pull-left finding-title">
                     {% if finding.publish_date %}
                         <th>Vuln Publish date</th>
                     {% endif %}
+                    {% if finding.planned_remediation_date %}
+                        <th>Planned Remediation Date</th>
```

Comment:
This area has already a lot of fields and this title needs much more space than the actual date. Can we just write "Planned Remediation"?

Reply:
Fine with me!